packages/spatial/src/input/systems/ClientInputSystem.tsx

```diff
@@ -399,7 +474,8 @@ const usePointerInputSources = () => {
     const pointerEnter = (event: PointerEvent) => {
       setComponent(emulatedInputSourceEntity, InputPointerComponent, {
         pointerId: event.pointerId,
-        canvasEntity: canvasEntity
+        canvasEntity: canvasEntity,
+        cameraEntity: canvasEntity //TODO likely want to double check this is done with a guaranteed camera (though it should exist)
```

Comment:
in what situation would we have a camera that is not the same as the canvas?

Reply:
removed, wasn't sure if we wanted to rely on canvasEntity having the camera reference or if there was a separate way in some cases. will use canvasEntity now